demo-shell/src/app/components/file-view/file-view.component.html

```diff
@@ -157,7 +157,7 @@ <h1>My custom toolbar</h1>
         </adf-info-drawer-tab>
 
         <adf-info-drawer-tab label="Versions">
-            <mat-card>
+            <mat-card appearance="outlined">
```

Comment:
this one will be removed before migration by https://github.com/Alfresco/alfresco-ng2-components/pull/9050